.github/workflows/tests.yml

```diff
@@ -1,6 +1,7 @@
 name: Run LAMMPS-Python tests
-
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
```

Comment:
does this behave differently than what was here before?

Reply:
No, it does not - I had to trigger a CI run. I can revert this.